.travis.yml

```diff
@@ -19,6 +19,10 @@ jobs:
       script:
         - cd conode
         - make docker
+        - docker run --rm -t -p 7770-7777:7770-7777 --name conode c4dt/conode:dev &
+        - make conode
+        - build/bcadmin latest --bc bc*cfg
```

Comment:
Concurrency issue, `docker run` may not have fully started yet. As ws is the main mean of communication and answers to HTTP, I used some curl loop in [c4dt/services](https://github.com/c4dt/services/blob/master/mk/service/backend.mk#L80).

Reply:
Oh my - this should still be ported completely to your services...

Reply:
Yup; hopefully, it will help to better know/understand what's happening. Maybe after finishing the CAS login?

Reply:
Added a `docker_wait` target to `conode/Makefile`